container_build_request.yaml

```diff
@@ -20,7 +20,10 @@ steps:
   args:
   - 'build'
   - '-t'
-  - 'gcr.io/${PROJECT_ID}/cos-gpu-installer:${_TAG}'
+  - 'gcr.io/${PROJECT_ID}/cos-gpu-installer:latest'
+  - '-t'
+  - 'gcr.io/${PROJECT_ID}/cos-gpu-installer:${TAG_NAME}'
```

Comment:
Where does TAG_NAME come from?

Reply:
The container builder will be configured to build and push a new docker image whenever a new tag is pushed to this repo. And TAG_NAME comes from the Github tag that triggers that build.
See https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values#using_default_substitutions